code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -306,7 +306,9 @@
 		if(!LAZYLEN(parts))
 			if(undismembermerable_limbs) //they have limbs we can't remove, and no parts we can, attack!
 				return ..()
-			to_chat(src, "<span class='cultlarge'>\"Bring [C.p_them()] to me.\"</span>")
+			C.Weaken(30)

```

Comment:
remove the Weaken(30) and actually fix the bug where failing to move with no limbs breaks pulls.